third_party/musl/src/starboard/sys/stat.c

```diff
@@ -37,11 +37,13 @@ int stat(const char *path, struct stat *file_info)
         return -1;
     }
 
-    file_info -> st_mode = 0;
+    // In SB_API_VERSION < 16, all files are opened with S_IRUSR | S_IWUSR.
+    // See http://shortn/_UxFowRzNXq.
```

Comment:
Just use the file path `starboard/shared/posix/impl/file_open.h` instead or url to internal code search.

Reply:
Done.